src/main/java/controller/BlackJackController.java

```diff
@@ -0,0 +1,124 @@
+package controller;
+
+import domain.cardtable.CardTable;
+import domain.deck.CardDeck;
+import domain.player.Name;
+import domain.player.Player;
+import domain.player.dealer.Dealer;
+import domain.player.dealer.DealerResult;
+import domain.player.participant.Participant;
+import domain.player.participant.ParticipantResult;
+import view.InputView;
+import view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.counting;
+
+public class BlackJackController {
+
+    public void run() {
+
+        final CardDeck cardDeck = CardDeck.shuffledFullCardDeck();
+        final CardTable cardTable = CardTable.readyToPlayBlackjack(cardDeck);
+
+        final List<Participant> participants = createParticipants();
+        final Dealer dealer = createDealer();
+        final List<Player> players = createPlayers(participants, dealer);
+
+        deal(cardTable, players);
+
+        printStateAfterDeal(participants, dealer);
+        hittingPlayer(cardTable, participants, dealer);
+        printStateAfterHit(participants, dealer);
```

Comment:
```suggestion
        deal(cardTable, players);
        printStateAfterDeal(participants, dealer);
        
        hittingPlayer(cardTable, participants, dealer);
        printStateAfterHit(participants, dealer);

```
요렇게 개행을 잡으면 의미 단위로 묶을 수 있을 것 같아요.